website/blog/2022-08-22-unit-testing-dbt-package.md

```diff
@@ -0,0 +1,200 @@
+---
+title: "An introduction to unit testing your dbt Packages"
```

Comment:
Just curious. Apparently, there is a spelling inconsistency on "dbt package(s)" or "dbt Package(s)". Which is better for the article?